AUTHORS

```diff
@@ -1,3 +1,4 @@
+James Moriarty <jamespaulmoriarty@gmail.com>
```

Comment:
Nit: this file should have entries that are sorted alphabetically

Reply:
👍